subprojects/packagefiles/liburing/meson.build

```diff
@@ -74,13 +88,19 @@ has_cxx = cpp.compiles(code, name: 'C++')
 has_ucontext = (cc.has_type('ucontext_t', prefix: '#include <ucontext.h>')
   and cc.has_function('makecontext', prefix: '#include <ucontext.h>'))
 
+has_memfd_create = cc.has_function('memfd_create',
+                                   args: ['-D_GNU_SOURCE'],
+                                   prefix: '#include <sys/mman.h>')
```

Comment:
But this commit is after the liburing-2.2 tag. Should we include the changes in our 2.2 wrap?

Reply:
Well, probably not. The newly added code is there in the release anyway, the define is needed to prevent conflicting definitions, but the newly added code was immediately dropped post-release, meh. :p